account_bank_statement_import_camt_details/models/account_bank_statement_line.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""Add process_camt method to account.bank.statement.import."""
+# © 2017 Compassion CH <http://www.compassion.ch>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, fields
+
+
+class AccountBankStatementLine(models.Model):
+    """Add process_camt method to account.bank.statement.import."""
+    _inherit = 'account.bank.statement.line'
+
+    partner_address = fields.Char()
+    partner_account = fields.Char()
+    partner_bic = fields.Char()
```

Comment:
I'm a bit doubtful about those two. Shouldn't they be encoded in https://github.com/OCA/OCB/blob/9.0/addons/account/models/account_bank_statement.py#L363